R/net.inputs.R

```diff
@@ -658,6 +658,18 @@ crosscheck.net <- function(x, param, init, control) {
         }
       }
 
+      # Checks that .m2 syntax update to .g2
+      m2.param <- grep("m2", names(param), value = TRUE)
```

Comment:
Isn't this in reverse of what it should be? Shouldn't this stop check be catching if the user passes in `.g2` parameters?

Reply:
I put that in there in case the user wasn't aware of the change as opposed to them neglecting to input the parameter. Can meld a helpful message into the original with something along the lines of "Note: mode language changed to group."

Reply:
ok, right, got it. resolving this.